test/PoolManager.t.sol

```diff
@@ -52,13 +54,15 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
         int24 tick,
         uint24 fee
     );
-    event ProtocolFeeUpdated(PoolId indexed id, uint16 protocolFee);
+    event ProtocolFeeUpdated(PoolId indexed id, uint24 protocolFee);
     event Transfer(
         address caller, address indexed sender, address indexed receiver, uint256 indexed id, uint256 amount
     );
 
     PoolEmptyUnlockTest emptyUnlockRouter;
 
+    uint24 constant MAX_FEE_BOTH_TOKENS = 10242500; // 2500 2500
```

Comment:
nit - i think this might be easier to read in hex. 0x9c49c4 - just because its clearly the same thing repeated twice
OR you could write it as like `(2500 << 12) & 2500`

Reply:
hex would be cheaper potentially? unless the optimizer does this for us